examples/execute/main.go

```diff
@@ -1,64 +1,51 @@
-package main
-
-import (
-	"context"
-	"encoding/json"
-	"fmt"
-	"io/ioutil"
-	"log"
-
-	"github.com/ShaileshSurya/hammer"
-)
-
-func handleErr(err error) {
-	if err != nil {
-		log.Fatal(err)
-	}
-}
-
-func main() {
-	// init client
-	client := hammer.New()
-
-	// build request
-	request, err := hammer.RequestBuilder().
-		Get().
-		WithURL("http://echo.jsontest.com/Greeting/hello/place/world").
-		WithContext(context.Background()).
-		WithHeaders("Accept", "application/json").
-		Build()
-	handleErr(err)
-
-	// Execute the request and manually handle response
-	resp, err := client.Execute(request)
-	handleErr(err)
-
-	body, err := ioutil.ReadAll(resp.Body)
-	handleErr(err)
-
-	type HelloWorld struct {
-		Greeting string
-		Place    string
-	}
-	var hw HelloWorld
-
-	err = json.Unmarshal(body, &hw)
-	handleErr(err)
-
-	fmt.Printf("%+v\n\n", hw)
-
-	// Execute the request and feed response body into an empty interface
-	m := make(map[string]interface{})
-	_ = client.ExecuteInto(request, &m)
-	handleErr(err)
-
-	fmt.Printf("%+v\n\n", m)
-
-	// Execute the request and feed response body into a struct
-	// (because a every struct implements interface{})
-	hw2 := HelloWorld{}
-	err = client.ExecuteInto(request, &hw2)
-	handleErr(err)
-
-	fmt.Printf("%+v\n\n", hw2)
-}
+package main
```

Comment:
The changes looks good to me. We might want to use the file path structure such as `examples\execute.go`

The user might run this with `go run execute.go`

With this file structure we wont need to create separate folder for each of the API. In future iteration we might want to add examples for RequestBuilder or in that case ResponseBuilder as well.

Reply:
@ZachPlunkett Looks like this change broke the CI pipeline. You were right, the proposed file structure by you was right. My bad. 

![image](https://user-images.githubusercontent.com/22809827/96751558-f9411880-13ea-11eb-948d-e803d5b62b0c.png)

Reply:
Fixed by reverting back to original structure